step-capstone/src/components/TimeLine.js

```diff
@@ -0,0 +1,166 @@
+import React, {useState, useEffect} from 'react';
+import '../styles/TimeLine.css';
+import DateFnsUtils from '@date-io/date-fns';
+import {
+ MuiPickersUtilsProvider,
+ KeyboardDatePicker,
+} from '@material-ui/pickers';
+import { isValid } from 'date-fns';
+
+function PickDisplayDate(props) {
+  const [displayDate, setDisplayDate] = useState(props.displayDate);
+
+  useEffect(() => {
+    setDisplayDate(props.displayDate);
+  }, [props.displayDate])
+
+  useEffect(() => {
+    props.onChangeDisplayDate(displayDate);
+  }, [displayDate])
+
+  return (
+    <MuiPickersUtilsProvider utils={DateFnsUtils} id="timeline">
+      <KeyboardDatePicker style={{    
+                              maxWidth: "141px",
+                              position: "relative",
+                              left: "15px"}}
+
+          variant="inline"
+          format="MM/dd/yyyy"
+          margin="normal"
+          id="displayDate"
+          label="Display Date"
+          value={displayDate}
+          onChange={setDisplayDate}
+      />
+    </MuiPickersUtilsProvider>
+  )
+}
+
+function ArrowLeft(props) {
+  const handleOnClick = () => {
+    var prevDay = new Date(props.displayDate);
+    prevDay.setDate(prevDay.getDate() - 1);
+    props.onChangeDisplayDate(prevDay);
+  }
+  return (
+    <div className="icon secondary chevron_left" onClick={handleOnClick}>‹</div>
+  )
+}
+
+function ArrowRight(props) {
+  const handleOnClick = () => {
+    var nextDay = new Date(props.displayDate);
+    nextDay.setDate(nextDay.getDate() + 1);
+    props.onChangeDisplayDate(nextDay);
+  }
+  return (
+    <div className="icon secondary chevron_left" style={{left: "15px"}} onClick={handleOnClick}>›</div>
+  )
+}
+
+
+function Header(props) {
+  return (
+    <div>
```

Comment:
Something to keep in mind perhaps for later changes: Using material UI components for UI consistency in all our components. For example, small things like button and text.

Reply:
Ok, I'll remember that